components/BlurhashCanvas.vue

```diff
@@ -37,15 +38,21 @@ export default Vue.extend({
   },
   methods: {
     draw() {
-      const pixels = decode(this.hash, this.width, this.height);
-      if (pixels) {
-        const ctx = (this.$refs.canvas as HTMLCanvasElement).getContext('2d');
-        const imageData = ctx?.createImageData(this.width, this.height);
-        if (imageData) {
+      const bhash_data = {
```

Comment:
No need to assign this since it's used only once and right on the next line, imo.